From b1a09529f3ab381494f3eecb53b2ce4adc5b55e9 Mon Sep 17 00:00:00 2001
From: Laurent Rineau <laurent.rineau@cgal.org>
Date: Tue, 17 Jan 2023 16:18:53 +0100
Subject: [PATCH] Add #include <cstdint> to compile with gcc13

The `vtkSEPReader` was introduced by MRs !4909 (from my former
collaborator Maxime) and !4938. Then it was highly modified by
!7516. The later MR is the one that introduced the uses of
`std::uint8_t` and `std::uint32_t`.

Those types needs the inclusion of `<cstdint>`.
---
 IO/Image/vtkSEPReader.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/IO/Image/vtkSEPReader.h b/IO/Image/vtkSEPReader.h
index 83d127a41e4..b9110780f26 100644
--- a/IO/Image/vtkSEPReader.h
+++ b/IO/Image/vtkSEPReader.h
@@ -25,8 +25,9 @@
 #include "vtkImageAlgorithm.h"
 #include "vtkNew.h" // for ivars
 
-#include <array>  // for std::array
-#include <string> // for std::string
+#include <array>   // for std::array
+#include <cstdint> // for std::uint8_t and std::uint32_t
+#include <string>  // for std::string
 
 namespace details
 {
-- 
GitLab

